src/core/Currency.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include "UmikoBot.h"
+
+class Currency {
+private:
+    int m_Cents;
```

Comment:
I think the idea is to store two integers: one whole (dollars) and one partition (cents)

Reply:
Fixed